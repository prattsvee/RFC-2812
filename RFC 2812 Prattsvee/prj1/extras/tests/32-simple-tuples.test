{ :a, 22 }

{:x, :y, :z}{}

{
33
} #{55}

