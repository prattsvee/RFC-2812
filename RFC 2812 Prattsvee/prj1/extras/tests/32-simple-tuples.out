[
  {
    "%k": "tuple",
    "%v": [
      {
        "%k": "atom",
        "%v": ":a"
      },
      {
        "%k": "int",
        "%v": 22
      }
    ]
  },
  {
    "%k": "tuple",
    "%v": [
      {
        "%k": "atom",
        "%v": ":x"
      },
      {
        "%k": "atom",
        "%v": ":y"
      },
      {
        "%k": "atom",
        "%v": ":z"
      }
    ]
  },
  {
    "%k": "tuple",
    "%v": []
  },
  {
    "%k": "tuple",
    "%v": [
      {
        "%k": "int",
        "%v": 33
      }
    ]
  }
]
